Improve Issue type "support" message (#19567)
PrestaShop/PrestaShop: .github/ISSUE_TEMPLATE/3_support_question.md

---
name: ⛔ I need help
about: For questions about using PrestaShop

---

We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.

If you have questions about using PrestaShop, you might find the answer here:
- User's guide - http://doc.prestashop.com/display/PS17/User+Guide
- Developer documentation - https://devdocs.prestashop.com/

> If you need help with your shop or third-party modules, please consider one of our support plans: https://www.prestashop.com/support

Reviewer: I have problem with our : who is `we` ? The Open Source Project ? The PrestaShop Company ? The Maintainers Team ?

Author: PS company

Reviewer: In this case, the `our` must be contextualized

Author: `$this->context->we` ?

Reviewer: :trollface: 

```suggestion
If you need help with your shop or third-party modules, please consider the PrestaShop Company support plans: https://www.prestashop.com/support
```

If you have questions about a PrestaShop's module/theme you have downloaded or bought on the PrestaShop Addons Marketplace, you can contact the module developer through the link provided on Addons module page.

Alternatively, you can also ask for help from other PrestaShop users:

> - Community forums: https://www.prestashop.com/forums/
> - Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
> - Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md

Reviewer: ```suggestion
- Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md
- Community forums: https://www.prestashop.com/forums/
- Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
```

---
name: ⛔ I need help
about: For questions about using PrestaShop

---

We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.

> If you have questions about using PrestaShop, you might find the answer in
> - PrestaShop user documentation http://doc.prestashop.com/display/PS17/User+Guide
> - PrestaShop developer documentation http://devdocs.prestashop.com/

Reviewer 2: ```suggestion

If you have questions about using PrestaShop, you might find the answer here:
- User's guide - http://doc.prestashop.com/display/PS17/User+Guide
- Developer documentation - https://devdocs.prestashop.com/

If you need help with your shop or third-party modules, please consider one of our support plans: https://www.prestashop.com/support
```

> You can also discuss with other PrestaShop users on
> - community forums (https://www.prestashop.com/forums/)
> - Stack Overflow (https://stackoverflow.com/questions/tagged/prestashop)
> - community Slack channel (https://github.com/PrestaShop/open-source/blob/master/slack/readme.md)
> 

Reviewer 2: ```suggestion
If you have questions about a PrestaShop's module/theme you have downloaded or bought on the PrestaShop Addons Marketplace, you can contact the module developer through the link provided on Addons module page.

Alternatively, you can also ask for help from other PrestaShop users:
- Community forums: https://www.prestashop.com/forums/
- Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
- Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md

```

> If you have questions about a PrestaShop you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.

Reviewer 3: ```suggestion
If you have questions about a PrestaShop's module/theme you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.
```

Isn't it better?

Author: Yes it is 😄

> 
> If you have questions about a PrestaShop's module/theme you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.
> 
> 
> If you need help with your shop or third-party modules, you can also refer to one of our support plans: https://www.prestashop.com/support
> 
> 

Reviewer 2: ```suggestion
```
